src/rpc/rawtransaction.cpp

```diff
@@ -1262,6 +1263,552 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
+            "            \"address\"               (string) Bitcoin address\n"
+            "            ,...\n"
+            "          ]\n"
+            "        }\n"
+            "      },\n"
+            "      \"partial_signatures\" : [             (array of json objects, optional)\n"
+            "        {\n"
+            "          \"pubkey\" : \"pubkey\",             (string) The public key this signature corresponds to\n"
+            "          \"signature\" : \"signature\",       (string) The signature\n"
+            "        }\n"
+            "        ,...\n"
+            "      ]\n"
+            "      \"sighash\" : \"type\",                  (string, optional) The sighash type recommended to be used\n"
+            "      \"redeem_script\" : {       (json object, optional)\n"
+            "          \"script\" : {           (json object)\n"
+            "            \"asm\":\"asm\",         (string) Script public key\n"
+            "            \"hex\":\"hex\",         (string) Hex encoded public key\n"
+            "            \"type\":\"type\",       (string) The output type\n"
+            "            \"reqSigs\": n,        (numeric) The required signatures\n"
+            "            \"addresses\": [       (json array of string)\n"
+            "              \"address\"           (string) Bitcoin address\n"
+            "               ,...\n"
+            "            ],\n"
+            "        }\n"
+            "      \"witness_script\" : {       (json object, optional)\n"
+            "          \"script\" : {            (json object)\n"
+            "            \"asm\" : \"asm\",        (string) The asm of the script\n"
+            "            \"hex\" : \"hex\",        (string) The hex of the script\n"
+            "          }\n"
+            "        }\n"
+            "       \"bip32_derivs\" : [          (array of json objects, optional)\n"
+            "          {\n"
+            "          \"pubkey\" : \"pubkey\",                     (string) The public key this path corresponds to\n"
+            "          \"master_fingerprint\" : \"fingerprint\"     (string) The fingerprint of the master key\n"
+            "          \"path\" : \"path\",                         (string) The path\n"
+            "          }\n"
+            "        }\n"
+            "        ,...\n"
+            "       ],\n"
+            "      \"unknown\" : {                (json object) The unknown global fields\n"
+            "        \"key\" : \"value\"            (key-value pair) An unknown key-value pair\n"
+            "         ...\n"
+            "      },\n"
+            "    }\n"
+            "    ,...\n"
+            "  ]\n"
+            "  \"outputs\" : [                 (array of json objects)\n"
+            "    {\n"
+            "      \"redeem_script\" : {       (json object, optional)\n"
+            "          \"script\" : {           (json object)\n"
```

Comment:
What is this "script" intermediate layer? I don't see that in the implementation (for this and for witness_script). Again, just "hex" and "asm" is sufficient here I think.

Reply:
Removed, that's a mistake